dashboard/app/models/pd/application/teacher1920_application.rb

```diff
@@ -353,13 +355,19 @@ def allow_sending_principal_email?
     end
 
     def allow_sending_principal_approval_teacher_reminder_email?
+      last_principal_approval_email = emails.where(email_type: 'principal_approval').order(:created_at).last
+      last_principal_approval_email_created_at = last_principal_approval_email&.created_at
+
       reminder_emails = emails.where(email_type: 'principal_approval_teacher_reminder')
 
       # Do we allow the cron job to send a reminder email to the teacher?
 
       # Only if we haven't already sent one.
       return false if reminder_emails.any?
 
+      # Only if we've sent at least one principal approval email before.
+      return false unless last_principal_approval_email_created_at
```

Comment:
It looks like we're not actually using the date, just whether there is one... and would we ever have a row without a `created_at`?

Put another way, could this just be
```rb
return false unless emails.where(email_type: 'principal_approval').exists?
```